AElf.Contracts.SideChain.Tests/AElf.OS/AElf.OS.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
use dot net strandard

AElf.Contracts.SideChain.Tests/AElf.OS/AElf.OS.csproj , path is not correct

Reply:
ok, thx. It is already fixed in my local.